### ISSUE TITLE
firebase_app_distribution plugin not working

### ISSUE THREAD

**Issue by thor-jc** [fastlane/fastlane#17419]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Similar to other closed issues I am having issues loading, updating or using the firebase_app_distribution plugin.  I've been through 2 different issues and there were several things to try and nothing seemed to work.  I'm hoping someone can find something obvious before I start from square 1 - if I can get back to that.

##### Command executed
bundle exec fastlane beta

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> thor@alloa android % bundle exec fastlane --verbose beta                  
[✔] 🚀 
DEBUG [2020-10-09 12:00:26.56]: Checking if there are any plugins that should be loaded...
ERROR [2020-10-09 12:00:26.56]: It seems like you wanted to load some plugins, however they couldn't be loaded
ERROR [2020-10-09 12:00:26.56]: Please follow the troubleshooting guide: https://docs.fastlane.tools/plugins/plugins-troubleshooting/
Successfully loaded Appfile at path '/Users/thor/alloatech/dev/at_base_lib/android/fastlane/Appfile'
- json_key_file: 'fastlane/conf/api-.json'
- package_name: ''
-------
INFO [2020-10-09 12:00:26.70]: -----------------------------
INFO [2020-10-09 12:00:26.70]: --- Step: update_fastlane ---
INFO [2020-10-09 12:00:26.70]: -----------------------------
INFO [2020-10-09 12:00:26.70]: Looking for updates for fastlane...
INFO [2020-10-09 12:00:29.36]: Nothing to update ✅
INFO [2020-10-09 12:00:29.36]: ------------------------------
INFO [2020-10-09 12:00:29.36]: --- Step: default_platform ---
INFO [2020-10-09 12:00:29.36]: ------------------------------
INFO [2020-10-09 12:00:29.36]: Driving the lane 'android beta' 🚀
INFO [2020-10-09 12:00:29.36]: -------------------------------
INFO [2020-10-09 12:00:29.36]: --- Step: number_of_commits ---
INFO [2020-10-09 12:00:29.36]: -------------------------------
INFO [2020-10-09 12:00:29.36]: $ git rev-parse HEAD
INFO [2020-10-09 12:00:29.37]: ▸ 18eb18f32047ca0e6978e908657668a8127cb224
INFO [2020-10-09 12:00:29.37]: $ git rev-list HEAD --count
INFO [2020-10-09 12:00:29.39]: ▸ 65
INFO [2020-10-09 12:00:29.39]: ----------------------------------
INFO [2020-10-09 12:00:29.39]: --- Step: flutter packages get ---
INFO [2020-10-09 12:00:29.39]: ----------------------------------
INFO [2020-10-09 12:00:29.39]: $ flutter packages get
INFO [2020-10-09 12:00:30.70]: ▸ Running "flutter pub get" in at_base_lib...                         0.6s
INFO [2020-10-09 12:00:31.22]: ---------------------------
INFO [2020-10-09 12:00:31.22]: --- Step: flutter clean ---
INFO [2020-10-09 12:00:31.22]: ---------------------------
INFO [2020-10-09 12:00:31.22]: $ flutter clean
INFO [2020-10-09 12:00:35.51]: ▸ Cleaning Xcode workspace...                                         3.6s
INFO [2020-10-09 12:00:35.84]: ▸ Deleting build...                                                  326ms
INFO [2020-10-09 12:00:35.85]: ▸ Deleting .dart_tool...                                               6ms
INFO [2020-10-09 12:00:35.85]: ▸ Deleting Generated.xcconfig...                                       0ms
INFO [2020-10-09 12:00:35.85]: ▸ Deleting flutter_export_environment.sh...                            0ms
INFO [2020-10-09 12:00:36.02]: -----------------------------------------------------------------------------------
INFO [2020-10-09 12:00:36.02]: --- Step: flutter build appbundle --target\=lib/main_qa.dart --build-number\=65 ---
INFO [2020-10-09 12:00:36.02]: -----------------------------------------------------------------------------------
INFO [2020-10-09 12:00:36.02]: $ flutter build appbundle --target\=lib/main_qa.dart --build-number\=65
INFO [2020-10-09 12:00:37.23]: ▸ Running "flutter pub get" in at_base_lib...                         0.7s
INFO [2020-10-09 12:01:22.08]: ▸ Running Gradle task 'bundleRelease'...
INFO [2020-10-09 12:01:22.08]: ▸ Note: Some input files use unchecked or unsafe operations.
INFO [2020-10-09 12:01:22.08]: ▸ Note: Recompile with -Xlint:unchecked for details.
INFO [2020-10-09 12:01:30.17]: ▸ Removed unused resources: Binary resource data reduced from 493KB to 463KB: Removed 6%
INFO [2020-10-09 12:01:34.44]: ▸ Running Gradle task 'bundleRelease'...                             56.7s
INFO [2020-10-09 12:01:34.45]: ▸ ✓ Built build/app/outputs/bundle/release/app-release.aab (22.8MB).
DEBUG [2020-10-09 12:01:34.63]: /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
Fastfile:34:in `block (2 levels) in parsing_binding'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/lane.rb:33:in `call'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/commands_generator.rb:352:in `run'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/bin/fastlane:23:in `<top (required)>'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/bin/fastlane:23:in `load'
/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/bin/fastlane:23:in `<top (required)>'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `load'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `kernel_load'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:28:in `run'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/cli.rb:476:in `exec'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/cli.rb:30:in `dispatch'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/cli.rb:24:in `start'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/exe/bundle:46:in `block in <top (required)>'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/lib/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/Library/Ruby/Gems/2.6.0/gems/bundler-2.1.4/exe/bundle:34:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'
WARN [2020-10-09 12:01:34.63]: Lane Context:
INFO [2020-10-09 12:01:34.63]: {:DEFAULT_PLATFORM=>:android, :PLATFORM_NAME=>:android, :LANE_NAME=>"android beta"}
ERROR [2020-10-09 12:01:34.63]: Could not find action, lane or variable 'firebase_app_distribution'. Check out the documentation for more details: https://docs.fastlane.tools/actions
INFO [2020-10-09 12:01:34.63]: Successfully generated documentation at path '/Users/thor/alloatech/dev/at_base_lib/android/fastlane/README.md'

+------+---------------------------------------------------------------+-------------+
|                                  fastlane summary                                  |
+------+---------------------------------------------------------------+-------------+
| Step | Action                                                        | Time (in s) |
+------+---------------------------------------------------------------+-------------+
| 1    | update_fastlane                                               | 2           |
| 2    | default_platform                                              | 0           |
| 3    | number_of_commits                                             | 0           |
| 4    | flutter packages get                                          | 1           |
| 5    | flutter clean                                                 | 4           |
| 6    | flutter build appbundle --target\=lib/main_qa.dart --build-nu | 58          |
+------+---------------------------------------------------------------+-------------+

ERROR [2020-10-09 12:01:34.65]: fastlane finished with errors
bundler: failed to load command: fastlane (/Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/bin/fastlane)
FastlaneCore::Interface::FastlaneError: [!] Could not find action, lane or variable 'firebase_app_distribution'. Check out the documentation for more details: https://docs.fastlane.tools/actions
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane_core/lib/fastlane_core/ui/interface.rb:141:in `user_error!'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/runner.rb:174:in `trigger_action_by_name'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
  Fastfile:34:in `block (2 levels) in parsing_binding'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/commands_generator.rb:352:in `run'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/gems/fastlane-2.162.0/bin/fastlane:23:in `<top (required)>'
  /Users/thor/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/bin/fastlane:23:in `load'
  /Users/thor/alloatech </pre>
</details>

### Environment

<details>
  <pre><details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                                     |
| --------------------------- | ------------------------------------------------------------------------- |
| OS                          | 10.15.6                                                                   |
| Ruby                        | 2.6.3                                                                     |
| Bundler?                    | true                                                                      |
| Git                         | git version 2.25.0                                                        |
| Installation Source         | ~/alloatech/dev/at_base_lib/android/vendor/bundle/ruby/2.6.0/bin/fastlane |
| Host                        | Mac OS X 10.15.6 (19G2021)                                                |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib            |
| OpenSSL Version             | LibreSSL 2.8.3                                                            |
| Is contained                | false                                                                     |
| Is homebrew                 | false                                                                     |
| Is installed via Fabric.app | false                                                                     |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                               |
| Xcode Version               | 12.0.1                                                                    |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
update_fastlane

default_platform(:android)

platform :android do
  desc "Runs all the tests"
  lane :test do
    gradle(task: "test")
  end



desc "Deploy a new beta build to Google Play"
  lane :beta do
    build_number = number_of_commits()
    Dir.chdir "../.." do
      sh("flutter", "packages", "get")
      sh("flutter", "clean")
      sh("flutter", "build", "appbundle", "--target=lib/main_qa.dart",  "--build-number=#{build_number}")
    end
    firebase_app_distribution(
            app: "1::android:",
            groups: "beta",
            release_notes: "test fastlane distro"
        )

    # upload_to_play_store(track: 'beta', aab: 'build/app/outputs/bundle/release/app.aab')
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
json_key_file("fastlane/conf/....") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
package_name("....") # e.g. com.krausefx.app

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.162.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| bundler                   | 2.1.4        |
| rake                      | 13.0.1       |
| CFPropertyList            | 3.0.2        |
| public_suffix             | 4.0.6        |
| addressable               | 2.7.0        |
| atomos                    | 0.1.3        |
| aws-eventstream           | 1.1.0        |
| aws-partitions            | 1.380.0      |
| aws-sigv4                 | 1.2.2        |
| jmespath                  | 1.4.0        |
| aws-sdk-core              | 3.109.1      |
| aws-sdk-kms               | 1.39.0       |
| aws-sdk-s3                | 1.83.0       |
| babosa                    | 1.0.4        |
| claide                    | 1.0.3        |
| colored                   | 1.2          |
| colored2                  | 3.1.2        |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| declarative               | 0.0.20       |
| declarative-option        | 0.1.0        |
| digest-crc                | 0.6.1        |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| dotenv                    | 2.7.6        |
| emoji_regex               | 3.0.0        |
| excon                     | 0.76.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 1.0.1        |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.7        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.2.0        |
| gh_inspector              | 1.1.3        |
| jwt                       | 2.2.2        |
| memoist                   | 0.16.2       |
| multi_json                | 1.15.0       |
| os                        | 1.1.1        |
| signet                    | 0.14.0       |
| googleauth                | 0.13.1       |
| httpclient                | 2.8.3        |
| mini_mime                 | 1.0.2        |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.3        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-core         | 1.5.0        |
| google-cloud-storage      | 1.29.1       |
| json                      | 2.3.1        |
| mini_magick               | 4.10.1       |
| plist                     | 3.5.0        |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| slack-notifier            | 2.3.2        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.8.1        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| word_wrap                 | 1.0.0        |
| nanaimo                   | 0.3.0        |
| xcodeproj                 | 1.19.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| xcpretty-travis-formatter | 1.0.0        |
</details>

*generated on:* **2020-10-09**
</details> </pre>
</details>

[Archive.zip](https://github.com/fastlane/fastlane/files/5356189/Archive.zip)

**Comment by rogerluan** [comment]
@thor-jc your logs say:

>It seems like you wanted to load some plugins, however they couldn't be loaded

And your `fastlane env` printed: 

>`No plugins Loaded`

This looks similar to https://github.com/fastlane/fastlane/issues/17192.
Francis and I discussed some solution [here](https://github.com/fastlane/fastlane/issues/17376#issuecomment-704339770), could you check it out to see if that's what's happening in your case as well? 😃

**Comment by thor-jc** [comment]
After reading that thread I tried setting my BUNDLE_GEMFILE env var but I'm not sure if that was the main part of that thread.   Since I followed many other threads I'm not sure I am in a good state.  E.g. I added a GEM_HOME bc when I tried to install the plugin I ran into permission errors and didn't want to run as sudo.

So my env for GEM looks like -> 
<details><pre>
thor@alloa at_base_lib % env | grep GEM                                                             
GEM_HOME=/Users/thor/.gems
BUNDLE_GEMFILE=/Users/thor/alloatech/dev/at_base_lib/android/Gemfile
</pre></details>

and I can see the plugin loaded -> 
<details><pre>
thor@alloa at_base_lib % ls -l ~/.gems/gems/         
total 0
drwxr-xr-x  24 thor  staff  768 Oct  9 11:13 fastlane-2.162.0
drwxr-xr-x   5 thor  staff  160 Oct 17 08:20 fastlane-plugin-firebase_app_distribution-0.2.4
</pre></details>

I can only guess that my env is pointing in too many directions and it is getting lost.

In case it help I think this is the only spot where I changed my profile related to this ... 

<details><pre>
export PATH="$PATH:$HOME/.gem/ruby/2.6.0/bin"
export GEM_HOME=~/.gems
export PATH="$PATH:$HOME/.gems/bin"
</pre></details>

Plus at some point I ran -> bundle install --path=vendor/bundle

Thanks for your time.

Looking forward to getting this up and running.

Regards,
John

**Comment by Gomes-89** [comment]
Thanks for the information [Omegle](https://www.omegle.ltd/)

**Comment by rogerluan** [comment]
> I can see the plugin loaded →

What's the output now when you run _fastlane_ ?

**Comment by thor-jc** [comment]
It still fails - I haven't changed anything since last post.

<details><pre>

thor@alloa android % bundle exec fastlane --verbose beta                                     
[✔] 🚀 
DEBUG [2020-11-07 09:34:36.70]: Checking if there are any plugins that should be loaded...
ERROR [2020-11-07 09:34:36.70]: It seems like you wanted to load some plugins, however they couldn't be loaded
ERROR [2020-11-07 09:34:36.70]: Please follow the troubleshooting guide: https://docs.fastlane.tools/plugins/plugins-troubleshooting/
Successfully loaded Appfile at path '/Users/thor/alloatech/dev/at_base_lib/android/fastlane/Appfile'
- json_key_file: 'fastlane/conf/api-6686450734489413895-926504-aa0ef93ddef7.json'
- package_name: 'com.alloatech.testlib'
-------
INFO [2020-11-07 09:34:36.82]: -----------------------------
INFO [2020-11-07 09:34:36.82]: --- Step: update_fastlane ---
INFO [2020-11-07 09:34:36.82]: -----------------------------
INFO [2020-11-07 09:34:36.82]: Looking for updates for fastlane...
INFO [2020-11-07 09:34:39.65]: Updating fastlane from 2.163.0 to 2.166.0... 🚀
INFO [2020-11-07 09:35:13.39]: Finished updating fastlane
INFO [2020-11-07 09:35:13.39]: Cleaning up old versions...
INFO [2020-11-07 09:35:13.39]: fastlane.tools successfully updated! I will now restart myself... 😴
[✔] 🚀 
[09:35:15]: It seems like you wanted to load some plugins, however they couldn't be loaded
[09:35:15]: Please follow the troubleshooting guide: https://docs.fastlane.tools/plugins/plugins-troubleshooting/
[09:35:15]: -----------------------------
[09:35:15]: --- Step: update_fastlane ---
[09:35:15]: -----------------------------
[09:35:15]: ------------------------------
[09:35:15]: --- Step: default_platform ---
[09:35:15]: ------------------------------
[09:35:15]: Driving the lane 'android beta' 🚀
[09:35:15]: -------------------------------
[09:35:15]: --- Step: number_of_commits ---
[09:35:15]: -------------------------------
[09:35:15]: $ git rev-parse HEAD
[09:35:15]: ▸ 21e7f3fa534bfa22e2c4281cdb982d09ab0b4bd2
[09:35:15]: $ git rev-list HEAD --count
[09:35:15]: ▸ 68
[09:35:15]: ----------------------------------
[09:35:15]: --- Step: flutter packages get ---
[09:35:15]: ----------------------------------
[09:35:15]: $ flutter packages get
[09:35:17]: ▸ Running "flutter pub get" in at_base_lib...                         1.1s
[09:35:18]: ---------------------------
[09:35:18]: --- Step: flutter clean ---
[09:35:18]: ---------------------------
[09:35:18]: $ flutter clean
[09:35:23]: ▸ Cleaning Xcode workspace...                                         4.3s
[09:35:24]: ▸ Deleting build...                                                1,138ms
[09:35:24]: ▸ Deleting .dart_tool...                                              18ms
[09:35:24]: ▸ Deleting Generated.xcconfig...                                       0ms
[09:35:24]: ▸ Deleting flutter_export_environment.sh...                            0ms
[09:35:24]: ▸ Deleting .flutter-plugins-dependencies...                            0ms
[09:35:24]: ▸ Deleting .flutter-plugins...                                         0ms
[09:35:24]: ---------------------------------------------------------------------------------------------
[09:35:24]: --- Step: flutter build apk --split-per-abi --target\=lib/main_qa.dart --build-number\=68 ---
[09:35:24]: ---------------------------------------------------------------------------------------------
[09:35:24]: $ flutter build apk --split-per-abi --target\=lib/main_qa.dart --build-number\=68
[09:35:26]: ▸ Running "flutter pub get" in at_base_lib...                         0.7s
[09:36:47]: ▸ Running Gradle task 'assembleRelease'...
[09:36:47]: ▸ Note: Some input files use unchecked or unsafe operations.
[09:36:47]: ▸ Note: Recompile with -Xlint:unchecked for details.
[09:37:17]: ▸ Removed unused resources: Binary resource data reduced from 703KB to 676KB: Removed 3%
[09:37:17]: ▸ Removed unused resources: Binary resource data reduced from 703KB to 676KB: Removed 3%
[09:37:17]: ▸ Removed unused resources: Binary resource data reduced from 703KB to 676KB: Removed 3%
[09:37:26]: ▸ Running Gradle task 'assembleRelease'...                          119.7s
[09:37:26]: ▸ ✓ Built build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk (9.3MB).
+------------------+--------------+
|          Lane Context           |
+------------------+--------------+
| DEFAULT_PLATFORM | android      |
| PLATFORM_NAME    | android      |
| LANE_NAME        | android beta |
+------------------+--------------+
[09:37:26]: Could not find action, lane or variable 'firebase_app_distribution'. Check out the documentation for more details: https://docs.fastlane.tools/actions

+------+------------------------------------+-------------+
|                    fastlane summary                     |
+------+------------------------------------+-------------+
| Step | Action                             | Time (in s) |
+------+------------------------------------+-------------+
| 1    | update_fastlane                    | 0           |
| 2    | default_platform                   | 0           |
| 3    | number_of_commits                  | 0           |
| 4    | flutter packages get               | 2           |
| 5    | flutter clean                      | 6           |
| 6    | flutter build apk --split-per-abi  | 121         |
|      | --target\=lib/main_qa.dart         |             |
+------+------------------------------------+-------------+

[09:37:26]: fastlane finished with errors

[!] Could not find action, lane or variable 'firebase_app_distribution'. Check out the documentation for more details: https://docs.fastlane.tools/actions

</pre></details>

**Comment by thor-jc** [comment]
> There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates.
> 
> Please make sure to update to the latest `fastlane` version and check if that solves the issue. Let us know if that works for you by adding a comment 👍
> 
> Friendly reminder: contributions are always welcome! Check out [CONTRIBUTING.md](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md) for more information on how to help with `fastlane` and feel free to tackle this issue yourself 💪

I did reply.  but maybe that is not directed at me.  I realize you are busy and this just seems odd.  I'll take any advise you have on next steps.

**Comment by lucastsantos** [comment]
I recently began to add Fastlane to some of my company apps. 
I've added the `firebase_app_distribution`plugin and worked flawlessly.

@thor-jc I would recommend to check the [integration steps](https://firebase.google.com/docs/app-distribution/android/distribute-fastlane) with Firebase App Distribution plugin.

Instead of
```
fastlane add_plugin firebase_app_distribution
```
use as:
```
bundle exec fastlane add_plugin firebase_app_distribution
```
Check if has created a file at `fastlane/Pluginfile` in your project.

Then, if everything is setup correctly, I think you might encounter another error. 😄 
Seems like Firebase App Distribution doesn't support App Bundle right now. 
You might need to swap your command to create an apk instead of aab.
